### ISSUE TITLE
How do you make this example work?

### ISSUE THREAD

**Issue by gfdsgf4ehy** [bigcommerce/checkout-sdk-js-example#30]
I've been working on trying to get this to display anything in my sandbox environment for over a week and it just doesn't work.  The best I've gotten it to do was to display nothing but the header and throw absolutely no errors.  I see it's downloaded the main.js in the browser's developer tools, but it just displays nothing.  

I have reinstalled node.js to different versions, manually modified code that threw errors.  Started over from scratch, added additional packages when error messages suggested I do so and quite frankly I'm completely frustrated.  Is there some specific version of Node.js I should be using?

**Comment by RobinOng** [comment]
Hi @gfdsgf4ehy, just to understand more about the issue you are facing. Did you enable the settings in Control Panel to use custom checkout?

I tested it with Node v10 and seems to be working.

**Comment by gfdsgf4ehy** [comment]
Yes I did enable customer checkout and it loads the initial file, but nothing happens but a header.

I'll try with Node v10 as at this point I don't remember what versions I've tried with
